Dynatrace/dynatrace-operator
# Change bracket type for labels and annotations in values.yaml

## Description

https://dt-rnd.atlassian.net/browse/K8S-10499

Adding a label using an umbrella helm chart that has the operator helm chart as a child. Due to labels being initalized as an array with labels: [] in [values.yaml](https://github.com/Dynatrace/dynatrace-operator/blob/16a6143e673fb73dd50d091c110c329e7d1686a1/config/helm/chart/default/values.yaml#L33), this causes an error when you try to override it.

`coalesce.go:289: warning: destination for dynatrace-operator-azure-app.dynatrace-operator.operator.labels is a table. Ignoring non-table value ([])`

There are many github issues where other charts did this and they got corrected - cert-manager is an example where they do this correctly: [cert-manager/deploy/charts/cert-manager/values.yaml at master · cert-manager/cert-manager](https://github.com/cert-manager/cert-manager/blob/master/deploy/charts/cert-manager/values.yaml#L22)

## How can this be tested?
Apply the helm chart with default values and check if it works still
Apply the helm chart with custom values set for annotations and labels in the values file
